src/background/utils.ts

```diff
@@ -114,6 +113,7 @@ export const redirectToWelcomeScreen = async (
   intent: InteractionIntent,
   errorCode?: ErrorCode,
 ): Promise<void> => {
+  const { OPEN_PAYMENTS_REDIRECT_URL } = await import('@/shared/defines');
```

Comment:
Otherwise importing this file in tests says `OPEN_PAYMENTS_REDIRECT_URL` is not defined.